bin/clean-jenkins-log

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cat the-build.log \
+    | sed \
+    -e 's|ha:////[a-zA-Z0-9/+][a-zA-Z0-9/+]*||' \
```

Comment:
If I'm reading the cleaned logs correctly, the `ha:////` thing is actually followed by a base64 string­ — which means _some_ lines also have a trailing `=` or `==`.

Reply:
Checking.

Reply:
Confirmed. Fixed with commit 42b293167d3ef6f65cdf8926e2387410cc0d58c5